src/RA_Leaderboard.h

```diff
@@ -8,41 +8,55 @@ class RA_Leaderboard
 {
 public:
     explicit RA_Leaderboard(_In_ const ra::LeaderboardID nLBID) noexcept;
-    virtual ~RA_Leaderboard() noexcept = default;
+    virtual ~RA_Leaderboard() noexcept             = default;
     RA_Leaderboard(const RA_Leaderboard&) noexcept = delete;
     RA_Leaderboard& operator=(const RA_Leaderboard&) noexcept = delete;
-    RA_Leaderboard(RA_Leaderboard&&) noexcept = default;
+    RA_Leaderboard(RA_Leaderboard&&) noexcept                 = default;
```

Comment:
Is this because of the "AlignConsecutiveAssignments" flag? Why are the first two not aligned with the last two, or the next one? Another vote to disable it.